apps/extension/src/injected-connection-manager.ts

```diff
@@ -17,11 +17,21 @@ import { ClientConnectionManager } from '@penumbra-zone/transport/src/chrome-run
  * handle an absence yet.
  */
 const services: typeof serviceTypeNames = [
-  'ibc.core.client.v1.Query',
   'penumbra.custody.v1alpha1.CustodyProtocolService',
+  'penumbra.view.v1alpha1.ViewProtocolService',
+
+  'ibc.core.client.v1.Query',
+
+  'penumbra.core.app.v1alpha1.QueryService',
+  'penumbra.core.component.chain.v1alpha1.QueryService',
+  'penumbra.core.component.compact_block.v1alpha1.QueryService',
+  'penumbra.core.component.dex.v1alpha1.QueryService',
   'penumbra.core.component.dex.v1alpha1.SimulationService',
+  'penumbra.core.component.governance.v1alpha1.QueryService',
+  'penumbra.core.component.sct.v1alpha1.QueryService',
+  'penumbra.core.component.shielded_pool.v1alpha1.QueryService',
+  'penumbra.core.component.stake.v1alpha1.QueryService',
   'penumbra.util.tendermint_proxy.v1alpha1.TendermintProxyService',
-  'penumbra.view.v1alpha1.ViewProtocolService',
 ];
```

Comment:
It's kinda a bummer we can't import these. Is the bundle _that_ much bigger if we do?

Reply:
Maybe you can document in code that this list needs to be in sync with the ones in the `types` package?

Reply:
using the type import and a literal list reduces the size of this bundle to 3.9kb, which i think is important because we inject and execute this script on every single page that the user visits. with some transport refactor like i mentioned this morning, we could soon remove this list entirely.

when importing the actual array from registry.ts, even as 'const', the injected script is over 30x larger and includes every complete service and method definition.

importing the type from registry.ts enforces sync with the type system.